picachu/tests/auth/test_auth.py

```diff
@@ -0,0 +1,36 @@
+import pytest
+from flask import url_for
+
+from picachu.config.auth_config import auth_username, auth_password
+
+
+def test_log_in_happy_path(flask_app):
+    data = {
+        'login': auth_username,
+        'password': auth_password,
+    }
+
+    response = flask_app.post(url_for('api.auth.log_in'), json=data)
+
+    assert response.status_code == 202
+
+
+@pytest.mark.parametrize(
+    'login, password',
+    [
+        ('Admin', 'admin'),
+        ('ADMIN', 'ADMIN'),
+        ('admin2', 'admin2'),
+        ('', 'admin2'),
```

Comment:
It would be cool if we broke these tests into groups, too. You can clearly see that you can take out tests for empty fields and tests for filled, but incorrect fields.

It is important to add tests:
1) correct login, empty password
2) empty login, correct password
3) two empty fields
4) correct login, incorrect password
5) incorrect login, correct password
6) incorrect login, incorrect password

Reply:
For all tests, it is important to check not only the status code, but also the scheme of the returned response.